### ISSUE TITLE
Clear "login" input when user successfully logs in

### ISSUE THREAD

**Issue by mememori** [HackerExperience/HEBorn#96]
Right now, if i login and then logout, the login form will still have my original input (including my password)

**Comment by renatomassaro** [comment]
Yep, Christian and I discussed about this yesterday

**Comment by kress95** [comment]
I made a temporary fix on PR #115, but it's not good enough (the sign-up fields aren't cleared) to fix this for real the `CoreModel` should become a union type:

```elm
type alias CoreModel =
    { route : Route
    , websocket : Websocket.Model
    }

type Route
    = Home ... -- should contain the landing page
    | Game ... -- should contain OS and Game
```

To make this possible, `config` should also be passed down to landing.
`Requests` should also get a minor change to allow it to receive any record containing config (instead of receiving game).

This is yet another (this time not so big) refactor we need to do, I'll updating the `Requests` API first as I was changing it already.

**Comment by kress95** [comment]
Another sidenote: logging out doesn't clear any models, so windows left open before logging out remain opened after logging in (even with another account).

This proposed refactor also fixes that.

**Comment by kress95** [comment]
Fixed by #131
